variables.tf

```diff
@@ -37,21 +37,27 @@ variable "enabled" {
   description = "A boolean to enable/disable resource creation"
 }
 
-variable "cache_enabled" {
-  type        = bool
-  default     = true
-  description = "If cache_enabled is true, create an S3 bucket for storing codebuild cache inside"
-}
-
 variable "cache_expiration_days" {
   default     = 7
-  description = "How many days should the build cache be kept"
+  description = "How many days should the build cache be kept. It only works when cache_type is 'S3'"
 }
 
 variable "cache_bucket_suffix_enabled" {
   type        = bool
   default     = true
-  description = "The cache bucket generates a random 13 character string to generate a unique bucket name. If set to false it uses terraform-null-label's id value"
+  description = "The cache bucket generates a random 13 character string to generate a unique bucket name. If set to false it uses terraform-null-label's id value. It only works when cache_type is 'S3"
+}
+
+variable "cache_type" {
+  type        = string
+  default     = "NO_CACHE"
+  description = "The type of storage that will be used for the AWS CodeBuild project cache. Valid values: NO_CACHE, LOCAL, and S3.  Defaults to NO_CACHE.  If cache_type is true, it will create an S3 bucket for storing codebuild cache inside"
```

Comment:
```suggestion
  description = "The type of storage that will be used for the AWS CodeBuild project cache. Valid values: NO_CACHE, LOCAL, and S3.  Defaults to NO_CACHE.  If cache_type is `S3`, it will create an S3 bucket for storing CodeBuild cache inside"
```

Reply:
Hello, sorry for the late response. Fixed it. Thank you for the review.